meta/duncrow-gmbh/opengraph/de.yml

```diff
@@ -0,0 +1,3 @@
+de:
+  title: OpenGraph-Protocol
+  description: Diese Erweiterung für Contao ermöglicht es, Daten für das OpenGraph-Protocol zu pflegen.
```

Comment:
Would you mind to add some keywords, too? See https://github.com/contao/package-metadata/blob/master/docs/de/README.md#beispiel-einer-yaml-sprachdatei